contrib/dracut/90zfs/module-setup.sh.in

```diff
@@ -5,7 +5,7 @@ check() {
 	[ "${1}" = "-d" ] && return 0
 
 	# Verify the zfs tool chain
-	for tool in "@sbindir@/zpool" "@sbindir@/zfs" "@sbindir@/mount.zfs" ; do
+	for tool in "@sbindir@/zpool" "@sbindir@/zfs" "@mounthelperdir@/mount.zfs" ; do
```

Comment:
I think you may need to add `-e 's,@mounthelperdir\@,$(mounthelperdir),g'` to the `sed` command in `contrib/dracut/90zfs/Makefile.am`.

Reply:
@dinatale2 you're right, thanks.